vendor/engines/split_accounts/app/models/split_accounts/split_account.rb

```diff
@@ -39,8 +41,9 @@ def extra_penny_count
 
     def duplicate_subaccounts?
       splits.map(&:subaccount_id)
-        .group_by { |e| e }
-        .any? { |k, v| v.size > 1 }
+        .group_by { |enum| enum }
```

Comment:
Would `subaccount_id` end up being a better variable name since that's essentially what it's grouping by?